src/vscode-dts/vscode.proposed.editorHover.d.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'vscode' {
+
+	/**
+	 * A hover represents additional information for a symbol or word. Hovers are
+	 * rendered in a tooltip-like widget.
+	 */
+	export class VerboseHover extends Hover {
```

Comment:
I guess you mean I can directly change the Hover class inside of the `vscode.d.ts` file. I'll do that.

Reply:
Do you mean I should be able to directly write a new Hover class inside of the propose d.ts file? Doing so throws the error `Duplicate identifier 'Hover'.`. Maybe this happens because Hover here is a class not an interface?

![Screenshot 2024-03-14 at 11 19 17](https://github.com/microsoft/vscode/assets/61460952/3b3b80c2-96cc-453f-beea-37db5d102190)

Or do you mean that I should directly change the Hover class inside of the `vscode.d.ts` file?

Reply:
Try `declare class Hover` or `declare export class Hover`. Maybe I'm wrong though and it does not work.

Reply:
Looks like I can directly change the `Hover` class, which I will do in the `vscode.d.ts` file.